params_bsd.go

```diff
@@ -0,0 +1,20 @@
+// +build openbsd freebsd
```

Comment:
Would you mind testing freebsd as well, or remove `freebsd` from here for now?

Reply:
Certainy, I will remove FreeBSD for now as I have not had the opportunity to test it.